src/chargeTask.hpp

```diff
@@ -22,13 +22,30 @@ class ChargeTask : public Task{
     void init(void);
     /**
      * @brief Charges the device, and exits to CLI on command
-    */
+     * 
+     * @return State of device: STATE_CLI or STATE_DEEP_SLEEP
```

Comment:
`AbstractTask::run` should be returning the next state of the device.

I would note that `ChargeTask::run` returns `STATE_CLI` when the CLI pattern is entered, or `STATE_DEEP_SLEEP` when the appropriate condition is met.

Reply:
Please resolve this issue before re-requesting review